pypeit/pypeit.py

```diff
@@ -482,16 +529,71 @@ def init_one_science(self, sci_ID, det):
         sci_image_files = self.fitstbl.find_frame_files('science', sci_ID=sci_ID)
         scidx = self.fitstbl.find_frames('science', sci_ID=sci_ID, index=True)[0]
         self.sciI = scienceimage.ScienceImage(self.spectrograph, sci_image_files, det=det,
+                                              binning = self.fitstbl['binning'][scidx],
```

Comment:
Not all instruments have ['binning'] set, e.g. NIRSPEC.
This will crash those instruments.